libsolidity/interface/StandardCompiler.cpp

```diff
@@ -316,6 +316,23 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 			return formatFatalError("JSONError", "Invalid input source specified.");
 	}
 
+	Json::Value const& smtlib2Responses = _input["auxiliaryInput.smtlib2responses"];
```

Comment:
Sorry I keep confusing you. I was ok with the two-levels of JSON, just suggested a different name for both levels.

Reply:
Haha ok